Gemfile

```diff
@@ -4,7 +4,8 @@ gem 'jekyll'
 gem 'go_script'
 
 group :jekyll_plugins do
-  gem 'guides_style_18f'
+  # https://github.com/18F/guides-style/pull/32
+  gem 'guides_style_18f', github: '18F/guides-style', branch: 'navtitle'
```

Comment:
This is meant to be temporary – we can wait to merge this until that one is done and a new release is cut if folks prefer.